src/__tests__/test-helper.ts

```diff
@@ -5,6 +5,19 @@ import {ConfigParams, LATEST_API_VERSION, Shopify} from '../base-types';
 import {MemorySessionStorage} from '../session/storage/memory';
 import {JwtPayload} from '../utils/types';
 import {getHMACKey} from '../utils/get-hmac-key';
+import {mockAdapter} from '../adapters/mock/adapter';
```

Comment:
I wonder if we could name this in a way that indicates it's just used to mock requests for tests, because the adapters aren't "supposed" to export anything. Maybe `mockHttpRequests` or some such?